platforms/core-execution/file-watching/src/integTest/groovy/org/gradle/internal/watch/DaemonReuseFileSystemWatchingIntegrationTest.groovy

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.watch
+
+import com.gradle.develocity.testing.annotations.LocalOnly
+
+@LocalOnly
+class DaemonReuseFileSystemWatchingIntegrationTest extends AbstractFileSystemWatchingIntegrationTest {
+
+    def setup() {
+        executer.requireDaemon()
+    }
+
+    def "can run consecutive builds in a single daemon without issues"() {
```

Comment:
Shouldn't `ChangesBetweenBuildsFileSystemWatchingIntegrationTest` cover this?

Reply:
Oh, yes, I'll remove this test